docs/ios/open-source/linking-embrace.md

```diff
@@ -63,3 +63,11 @@ you will need to install an additional pod:
 pod 'EmbraceCrashlyticsSupport'
 ```
 :::
+
+
+### Known Issues
+
+* We rely on [OpenTelemetry-Swift](https://github.com/open-telemetry/opentelemetry-swift/) which as of today does not
```

Comment:
@ArielDemarco let me know if this accurately describes the issue. I also wasn't sure if this was specific to Cocoapods or relevant to the SPM section as well

Reply:
AFAIK the problem happens with both dependency management tools.
Description is clear enough though 👍

Reply:
sounds good, I just changed it to a H2 then so that it is at the same level as the other headers